templates/galaxy/config/user_preferences_extra_conf.yml.j2

```diff
@@ -90,4 +90,22 @@ preferences:
               label: App Password
               type: password
               required: False
+    
+    xnat_account:
```

Comment:
Could you please surround this block with {% if %} tags in the same way the the chatgpt block is surrounded earlier in the file? This way it will only be on dev while it's a dev tool

Reply:
@cat-bro I just added the (% if %} to the user_preferences.